CONTRIBUTING.md

```diff
@@ -3,6 +3,8 @@
 * Use CamelCase variables
 * Tabs, not spaces, with tab width set to 4
 * Follow https://www.autoitscript.com/wiki/Best_coding_practices
+* Tidy code with Tidy.exe before you create PR
+* Use Au3Check.exe for pre check code before you create PR
```

Comment:
This is already in the "best coding practices" link

Reply:
oops. checked. 

Almost true.
I mean there is mentioned to use Au3Check but not about Tidy.exe

What you think about ?

Reply:
I think that tidy step should be first step in pull request guidelines section and get rid of au3check

Reply:
Nah Au3Check should be included or it'll main au3check on the main script

Reply:
I still think these should be added as first in pr guidelines, for example:
Run Tidy and Au3check on your code to make sure that it is formatted properly.

Reply:
But then it doesn't look *NICE* as a list 😆

Reply:
eh ok